calendar-service/calendar-service.sh

```diff
@@ -33,25 +33,24 @@ start() {
 
   if [ "$pid" -ne 0 ] ; then
     echo "Calendar service is already started"
-    exit 1
+  else
+    # If the application isn't running, starts it
+    echo " *** Starting Calendar Service *** "
+
+    #copy scheduler users to radicale server
+    loginFile="$cwd/config/authentication/login.cfg"
+    rm -rf ~/.config/radicale/users
+    touch ~/.config/radicale/users
+
+    while IFS=':' read -r user password
+    do
+        htpasswd -bs ~/.config/radicale/users "$user" "$password"    
```

Comment:
Are you sure that _htpasswd_ is a common command? It seems not to be the case on my distribution. I checked with a default Docker image for Ubuntu and same problem.

You could add a check and suggest installation of the required packages as it is done in _install.sh_ script:

https://github.com/ow2-proactive/scheduling/blob/master/tools/install.sh#L90

otherwise invoking your script may fail. Or at least explain what are the required packages in the documentation.